ui/pages/settings/settings.component.js

```diff
@@ -69,6 +69,7 @@ class SettingsPage extends PureComponent {
     isPopup: PropTypes.bool,
     mostRecentOverviewPage: PropTypes.string.isRequired,
     pathnameI18nKey: PropTypes.string,
+    remoteFeatureFlags: PropTypes.object,
```

Comment:
This should also be marked as required, since the `InfoTab` needs it

```suggestion
    remoteFeatureFlags: PropTypes.object.isRequired,
```

Reply:
Good point ~ addressed 545943e3a603e160da198d220ddd14c21213b347